### PR TITLE
Fix/bump

### PR DESCRIPTION
**Pull request by cloudlovely**
## what
* Added key-vault expiry date.
* Fixed dependabot bumps.

## why
* Need to add key-vault expiry date.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -125,12 +125,13 @@ resource "azurerm_role_assignment" "identity_assigned" {
...
-  depends_on   = [azurerm_role_assignment.identity_assigned]
-  count        = var.enabled && var.default_enabled == false ? 1 : 0
-  name         = format("storage-%s-cmk-key", module.labels.id)
-  key_vault_id = var.key_vault_id
-  key_type     = "RSA"
-  key_size     = 2048
+  depends_on      = [azurerm_role_assignment.identity_assigned]
+  count           = var.enabled && var.default_enabled == false ? 1 : 0
+  name            = format("storage-%s-cmk-key", module.labels.id)
+  expiration_date = "2023-12-31T18:29:59Z"
```
Comment (13archit) [inline main.tf:131]: Pass Variable for this.

File: main.tf [hunk 2]
```diff
@@ -139,6 +140,14 @@ resource "azurerm_key_vault_key" "kvkey" {
     "verify",
     "wrapKey",
   ]
+  rotation_policy {
```
Comment (13archit) [inline main.tf:143]: Can we make is dynamic?

Comment (cloudlovely) [reply main.tf:143]: Yes, we can set it dynamic but automatic key rotation policy is better to use to rotate the key at specific time interval.

Comment (cloudlovely) [reply main.tf:143]: Yes, we can set it dynamic but automatic key rotation policy is better to use to rotate the key at specific time interval.

Comment (13archit) [reply main.tf:143]: By default we can set it to true

Comment (cloudlovely) [reply main.tf:143]: what?

**Pull request merged**